pennylane_lightning/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.24.0-dev15"
+__version__ = "0.24.0"
```

Comment:
```suggestion
__version__ = "0.24.0-rc0"
```

Reply:
The RC0 removal is the last step before launching. We should keep the tag until then.